python/pyspark/context.py

```diff
@@ -152,10 +152,10 @@ class SparkContext(object):
     """
 
     # set assignment ignore temporarily to prevent errors from other files
```

Comment:
So, then I think we don't need this comment anymore ?

Reply:
make sense. I will revise it.